src/lint/collect-spelling-diagnostics.ts

```diff
@@ -0,0 +1,60 @@
+import type { LintingError } from './algorithm-error-reporter-type';
+
+import { offsetToLineAndColumn } from './utils';
+
+let ruleId = 'spelling';
+
+// Note that these will be composed, so cannot contain backreferences
+let matchers = [
+  {
+    pattern: /\*this\* object/giu,
+    message: 'Prefer "the *this* value"',
```

Comment:
I think the intent here (with `the` in the message but not in the pattern) is clear, so I don't object, but I will note that there are a few instances of `a *this* value` in the spec.

Reply:
Removed the `the`.